src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsReconciliationService.kt

```diff
@@ -59,7 +59,24 @@ class AdjudicationsReconciliationService(
 
     val nomisAdaSummary: AdaSummary = nomisSummary.toAdaSummary()
     val dpsAdaSummary: AdaSummary = dpsAdjudications.toAdaSummary()
-    return if (nomisAdaSummary != dpsAdaSummary) MismatchAdjudicationAdaPunishments(prisonerId = prisonerId, dpsAdas = dpsAdaSummary, nomisAda = nomisAdaSummary) else null
+    return if (nomisAdaSummary != dpsAdaSummary) {
+      MismatchAdjudicationAdaPunishments(prisonerId = prisonerId, dpsAdas = dpsAdaSummary, nomisAda = nomisAdaSummary).also { mismatch ->
+        log.info("Adjudications Mismatch found  $mismatch")
+        telemetryClient.trackEvent(
+          "adjudication-reports-reconciliation-mismatch",
+          mapOf(
+            "offenderNo" to mismatch.prisonerId.offenderNo,
+            "bookingId" to mismatch.prisonerId.bookingId.toString(),
+            "nomisAdaCount" to (mismatch.nomisAda.count.toString()),
+            "dpsAdaCount" to (mismatch.dpsAdas.count.toString()),
+            "nomisAdaDays" to (mismatch.nomisAda.days.toString()),
+            "dpsAdaDays" to (mismatch.dpsAdas.days.toString()),
+          ),
```

Comment:
I might endup putting more information in this, like adjudication numbers - but for stick to the basics